python/labbox_ephys/api/_workersession.py

```diff
@@ -273,3 +271,24 @@ def _is_jsonable(x) -> bool:
         return True
     except:
         return False
+
+def _feed_id_from_uri(uri: str):
+    a = uri.split('/')
+    return a[2]
+
+def _subfeed_hash_from_name(subfeed_name: Union[str, dict]):
+    if isinstance(subfeed_name, str):
+        if subfeed_name.startswith('~'):
+            return subfeed_name[1:]
```

Comment:
To be clear, any name starting with a tilde is assumed to be actually a hash. Is there a capacity for the user to pick their own name for a subfeed? If so, we should probably warn them not to use this symbol at the beginning of the feed id (which someone might be tempted to do, since it's common as a shorthand for one's home directory).

Reply:
yes, I created an issue for this.